util/ntp/ntp.go

```diff
@@ -0,0 +1,134 @@
+package ntp
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"math"
+	"sync"
+	"time"
+
+	"github.com/beevik/ntp"
+	"github.com/pactus-project/pactus/util/logger"
+)
+
+const (
+	maxClockOffset = time.Duration(math.MinInt64)
+)
+
+type Checker struct {
+	lock   sync.RWMutex
+	ctx    context.Context
+	cancel func()
+
+	threshold  time.Duration
+	offset     time.Duration
+	interval   time.Duration
+	logger     *logger.SubLogger
```

Comment:
Please avoid defining sub logger for small modules.
Here we can use the global logger.

Reply:
we should have a consistent approach to this